tests/test_develop.py

```diff
@@ -1,19 +1,33 @@
 
-import importlib
-import os
-import os.path as osp
+
+import pytest
 import subprocess
 import shutil
+import pathlib
 import sys
 
 
-def test_develop(package_dir):
+data_files_specs = [
+    ("jupyter-packaging-test", "share", "test.txt"),
```

Comment:
Ideally we would be able to specify this as `("jupyter-packaging-test", "share", "**.*")` I think

Reply:
Right. I guess I really need two things here... i) data_files_spec (represented as globs) and ii) the actual filepaths to create for the test.

I was using this one argument for both the globbed spec and the argument used to create these files on disk (which need to be more specific). I'll update

Reply:
Ok, this should be addressed in the latest commit.